### PR TITLE
proper error response for a document not found:

### PR DESCRIPTION
**Pull request by FBRTMaka**
@DanielJMaher ,  This is a document that couldn't be found.  Adding a catch to handle a CMIS exception to return a 404 instead of a 500

### CODE REVIEW

File: ooiservices/app/alfresco/routes.py [hunk 1]
```diff
@@ -83,6 +84,12 @@ def get_doc():
         results,cruise = alf.make_alfresco_cruise_query(array,cruise)
 
         #results = alf.make_alfresco_query(query)
+    except cmislib.exceptions.ObjectNotFoundException as e:
+        print e
```
Comment (oceanzus) [inline ooiservices/app/alfresco/routes.py]: Can we log instead of print please?

File: ooiservices/app/alfresco/routes.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 from flask import jsonify, make_response, request
 import json
+import cmislib
```
Comment (oceanzus) [inline ooiservices/app/alfresco/routes.py]: Does this require a change to the requirements file?

File: ooiservices/app/alfresco/routes.py [hunk 3]
```diff
@@ -83,6 +84,12 @@ def get_doc():
...
 
         #results = alf.make_alfresco_query(query)
+    except cmislib.exceptions.ObjectNotFoundException as e:
+        print e
+        error_response = {'error': 'Document not found,' +
+            ' cannot create download link.', 'status_code': 404}
+        response = make_response(jsonify(error_response), 404)
+        return response, response.status_code
     except Exception as e:
         print e
```
Comment (oceanzus) [inline ooiservices/app/alfresco/routes.py]: log it

File: ooiservices/app/alfresco/routes.py (hunk 1 above)
Comment (FBRTMaka) [inline ooiservices/app/alfresco/routes.py]: absolutely!

**Comment by oceanzus** [comment]
@DanielJMaher @FBRTMaka Looks good to me.   When should this get merged in?

**Comment by DanielJMaher** [comment]
Fire when ready.

**Pull request merged**